docs/installation.md

```diff
@@ -0,0 +1,365 @@
+# Installation
+
+This document provides detailed instructions on how to install the project. Follow these steps to set up the project on your local system.
+
+Instructions for Windows should theoretically work but we have only tested this project to work on -nix systems. As such, we are unable to verify any discrepancies on Windows, and there could be slight variations in your setup.
```

Comment:
Disclaimer is added here, hope that's okay? @illusional

Reply:
Oh yep, could we move it into the Windows section?

Reply:
Easy done!